src/app/features/quest/quest-preview/quest-preview.service.ts

```diff
@@ -304,4 +305,54 @@ export class QuestPreviewService {
     return Number(value);
   }
 
+  getObjText(field: string | number) {
+    return this.questTemplate[`ObjectiveText${field}`];
```

Comment:
as the `ObjectiveText*` are a fixed amount of fields, isn't it better to create just one getter per field?